slides/_naming_for_clarity.qmd

```diff
@@ -1,4 +1,4 @@
-## Naming For Clarity
+## Naming For Clarity {.smaller}
```

Comment:
When I render the slides the list below runs off the bottom of the slide.
Consider splitting across 2 slides or reducing text size.

The latter can be done by bracketing the list as follows:
```
::: {style="font-size: 50%;"}

- Item 1
- Item 2
- Item 3

:::
```